packages/app/src/debug/DebugCancelledAlert.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Alert
+    :title="t('debugPage.manuallyCancelled')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="flex items-center">
+      <div>{{ t('debugPage.specsSkipped', {n: totalSpecs, totalSkippedSpecs}) }}</div>
+      <div
+        v-if="cancelledByEmail && cancelledByFullName"
+        class="rounded-full font-semibold bg-orange-500 h-3px mx-10px w-3px"
+      />
+      <div
+        v-if="cancelledByEmail && cancelledByFullName"
+        class="flex items-center"
+      >
+        <UserAvatar
+          :email="cancelledByEmail"
+          class="h-20px mr-7px w-20px"
+          data-cy="cancelled-by-user-avatar"
+        />
+        <div>{{ cancelledByFullName }}</div>
+      </div>
```

Comment:
Rather than doing the two side-by-side `v-if` with the same conditional could we wrap with a single conditional `template`?

```
<template v-if="cancelledByEmail && cancelledByFullName">
   <div ... />
   <div ... />
</template>
```

Reply:
Nice, I didn't think of this!